src/controller.rs

```diff
@@ -289,9 +290,12 @@ describe! controller {
     before_each {
         use stubs::service::ServiceStub;
         use tls::TlsOption;
+        use profile_service::ProfilePath;
 
         let service = ServiceStub;
-        let controller = FoxBox::new(false, Some("foxbox".to_owned()), 1234, 5678, TlsOption::Disabled);
+        let controller = FoxBox::new(
+          false, Some("foxbox".to_owned()), 1234, 5678,
+                      TlsOption::Disabled, ProfilePath::Default);
```

Comment:
I see it in Servo
https://github.com/servo/webrender/blob/2a71ccb0249601b25ef156f9a585a001c8d606cb/src/render_backend.rs#L152-L158